miso-web/src/main/webapp/styles/style.css

```diff
@@ -1848,90 +1848,6 @@ button.disabled {
   opacity: .4;
 }
 
- .portlet {

```

Comment:
This CSS styling was introduced in an earlier commit by me, and I am now deliberately removing it.